flink-libraries/flink-table/src/main/scala/org/apache/flink/table/runtime/aggregate/AggregateUtil.scala

```diff
@@ -89,6 +95,18 @@ object AggregateUtil {
         aggregationStateType)
     }
   }
+ 
+  def getLowerBoundary(
+      constants: ImmutableList[RexLiteral],
+      lowerBound: RexWindowBound,
+      input: RelNode): Int = {
+    val ref: RexInputRef = lowerBound.getOffset.asInstanceOf[RexInputRef]
+    val index:Int = ref.getIndex
+    val count: Int = input.getRowType.getFieldCount
```

Comment:
Can you explain more why we using `input.getRowType.getFieldCount` to calculation the LowerBoundary?